superchain/implementations/networks/sepolia.yaml

```diff
@@ -10,7 +10,6 @@ optimism_mintable_erc20_factory:
   1.9.0: 0xE01efbeb1089D1d1dB9c6c8b135C934C0734c846
 optimism_portal:
   2.5.0: 0x2D778797049FE9259d947D1ED8e5442226dFB589
-  2.5.1: 0xFF6Ce11Cd5094569E4Fc67C5b508F22754dA0667
```

Comment:
are we sure we want to delete this?

Reply:
I confirmed it was added in the PR that added the seal chains, and we haven't added any chains since then https://github.com/ethereum-optimism/superchain-registry/blame/main/superchain/implementations/networks/sepolia.yaml